game/game/api/world.py

```diff
@@ -28,6 +31,13 @@ async def enter_world(request: GameRequest):
 
     request.session.send_packet(game.packets.ItemList(items=character.inventory.items))
 
+    welcome = SystemMessage(type=WELCOME_TO_LINEAGE)
+    WORLD.send_sys_message(character, welcome)
+
+    msg_values = [ZoneName(x=-71338, y=258271, z=-3104)]
+    territory = SystemMessage(type=LOC_TI_S1_S2_S3, values=msg_values)
```

Comment:
why is this static?

Reply:
sorry, it was for testing purpose, fortgot to remove.

Reply:
it was removed in [](https://github.com/Yurzs/L2py/pull/23/commits/ca82c2bab8cbc72cb5d33d3bc11de984b2b6dd27)